manual/manual/foreword.etex

```diff
@@ -45,25 +45,24 @@ The OCaml system is open source and can be freely
 redistributed.  See the file "LICENSE" in the distribution for
 licensing information.
 
-The present documentation is copyright \copyright\ \number\year\
+The OCaml documentation and user's manual is
+copyright \copyright\ \number\year\
 Institut National de Recherche en Informatique et en
-Automatique (INRIA).  The OCaml documentation and user's
-manual may be reproduced and distributed in whole or
-in part, subject to the following conditions:
-\begin{itemize}
-\item The copyright notice above and this permission notice must be
-preserved complete on all complete or partial copies.
-\item Any translation or derivative work of the OCaml
-documentation and user's manual must be approved by the authors in
-writing before distribution.
-\item If you distribute the OCaml
-documentation and user's manual in part, instructions for obtaining
-the complete version of this manual must be included, and a
-means for obtaining a complete version provided.
-\item Small portions may be reproduced as illustrations for reviews or
-quotes in other works without this permission notice if proper
-citation is given.
-\end{itemize}
+Automatique (INRIA).
+
+\begin{latexonly}
+The OCaml documentation and user's manual is licensed under a Creative
+Commons Attribution-ShareAlike 4.0 International License (CC BY-SA
+4.0), \url{https://creativecommons.org/licenses/by-sa/4.0/}.
+\end{latexonly}
+
+\begin{htmlonly}
+\begin{rawhtml}
+<a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png"></a>
```

Comment:
Small nitpicking question, the creative commons logo is displayed inline as
```
┌───────┐ 
│ CC    │
└───────┘ The OCaml documentation and·········
··············································
```
Do we want to keep it inline or should we make it a block element, i.e.
```
┌───────┐ 
│ CC    │
└───────┘
The OCaml documentation and···················
··············································
```

Reply:
My HTML skills are as weak as my design skills :-)  Feel free to improve the look by pushing directly to the branch for this PR.

Reply:
Let's defer the styling decision then: I will push a change that adds both an identifier to the logo and the current style to the css.  The precise logo position can be tuned afterwards.

Reply:
After some A/B testing with @gasche , I pushed the id change with some basic styling to get
```
┌───────┐ 
│ CC    │  The OCaml documentation and·········
│       │ ·····································
└───────┘  
````

Reply:
Perfect!  Thanks for the styling effort.  Would you please merge and cherry-pick to 4.10?  Otherwise I'll do it when I can, but it may take a couple of days.